modules/calib3d/include/opencv2/calib3d.hpp

```diff
@@ -526,6 +532,25 @@ enum HandEyeCalibrationMethod
     CALIB_HAND_EYE_DANIILIDIS   = 4  //!< Hand-Eye Calibration Using Dual Quaternions @cite Daniilidis98
 };
 
+enum SamplingMethod { Uniform, ProgressiveNAPSAC, Napsac, Prosac, Evsac };
```

Comment:
please, use "all-capitals" notation; also, Uniform (and GC, SIGMA, Flann, Grid below) are too generic names;
please, use `SAMPLING_UNIFORM, SAMPLING_PROGRESSIVE_NAPSAC, ...`,
`LOCAL_OPTIM_NULL, LOCAL_OPTIM_GC, ...`
etc. that is, use some prefix in each enumeration.

Reply:
done